CONTRIBUTING.md

```diff
@@ -34,22 +34,16 @@ tests for you.
 cargo test --features derive
 ```
 
-##### In the [`test_suite/deps`] directory
-
-```sh
-# This is a prerequisite for running the full test suite
-cargo clean && cargo update && cargo build
-```
-
 ##### In the [`test_suite`] directory
 
 ```sh
 # Run the full test suite, including tests of unstable functionality
-cargo test --features unstable
+cargo +nightly test --features unstable
 ```
 
+Note that this test suite currently only supports running on a nightly compiler.
```

Comment:
Maybe add a `cfg` that restrict running tests that requires nightly only on nightly compiler?